src/Auth/ResetPassword/ResetPasswordController.php

```diff
@@ -136,7 +136,7 @@ public function resetPassword()
     public function generateResetToken()
     {
         // Retrieve data
-        $email = Input::get('email');
+        $email = urldecode(Input::get('email'));
```

Comment:
urldecode will convert a `+` to a space. So any email address with a `+` may cause the check on `filter_var($email, FILTER_VALIDATE_EMAIL)` to fail. Has this case been tested?